css/main.css

```diff
@@ -52,3 +87,96 @@ td.drawn {
 a.btnStack:hover {
     background-color: #000;
 }
+
+.calledBoxContainer {
+    width: 50px;
+    display: inline-flex;
+    justify-content: flex-end;
+}
+
+.board {
```

Comment:
Wouldn't this be a perfect case for css grid rather than flexbox?

Reply:
Yes and no. From when I've tried CSS grid, it was more awkward/more work than I had hoped to make it do what I was expecting.

It's probably most beneficial if contents are going to jump between grid row/column coordinates frequently/ever because it would definitely make that trivial but for a static layout I don't feel like it's much different either way.